win32/src/win32apimodule.cpp

```diff
@@ -6263,17 +6260,14 @@ PYWIN_MODULE_INIT_FUNC(win32api)
     PyModule_AddIntConstant(module, "VS_FF_PRIVATEBUILD", VS_FF_PRIVATEBUILD);
     PyModule_AddIntConstant(module, "VS_FF_SPECIALBUILD", VS_FF_SPECIALBUILD);
 
-    HMODULE hmodule = GetModuleHandle(TEXT("secur32.dll"));
-    if (hmodule == NULL)
-        hmodule = LoadLibrary(TEXT("secur32.dll"));
+    // clang-format off
```

Comment:
I think those format comments are also redundant (I'm guessing another leftover?). Running clang-format locally I'm not seeing any difference

 ```suggestion
```